core/src/main/java/aima/core/environment/support/ProblemFactory.java

```diff
@@ -51,7 +52,24 @@
 				Map2DFunctionFactory.getStepCostFunction(simplifidRoadMapOfPartOfRomania));
 	}
 
-	public static Problem<String, VEWorldState> getSimpleVacuumWorldProblem(String inInitialLocation,
+  public static BidirectionalProblem<GoAction, InState> romaniaRoadMapProblem(

```

Comment:
I would name this method:

getSimplifiedRoadMapOfPartOfRomaniaBidirectionalProblem()

to keep it consistent with the method defined above it.